### ISSUE TITLE
Can't reproduce top-1 85% results

### ISSUE THREAD

**Issue by Xyand** [zhang-can/ECO-pytorch#13]
Hi,

While running ECO-lite 4F I couldn't reach the reported results of top-1 85%. Instead got around 68%. Is there anything extra that needs to be done?

To my knowledge the only thing I'm doing differently is using a newer version of PyTorch. 

Thanks

**Comment by Xyand** [comment]
Problem was due to faulty split file generation. The number of frames was always set to 0 instead of the real number of frames. Probably causing the training to use a single image only.

**Comment by Xyand** [comment]
Ok. After fixing the bug I mentioned above I managed to get up to 80.5% top-1 accuracy after 80 epoch with the parameters you provided. Quite far from the steep graph you posted on ECO-efficient-video-understanding. 

Do you have an idea what could cause this difference? Btw, top-1 accuracy on train (split-1) reaches 97.5%

![image](https://user-images.githubusercontent.com/1929813/44146784-f5bcea0c-a099-11e8-95c0-318afb0a3a81.png)

**Comment by zhang-can** [comment]
Hi @Xyand 
Could you please provide me with your log file? Then I will have a check.

Note that this repo is not finished yet and the current pretrained model is not well trained! I will update as soon as possible.

**Comment by Xyand** [comment]
Thanks for the response. Here is a link for the log file of one of the training setups:
https://drive.google.com/file/d/1dvL3cnuOYZ2m53aIUoqut9HIZwGwopnu/view?usp=sharing

Waiting for your updates 👍

**Comment by Xyand** [comment]
Thanks. Got to 84.2% using pytorch 0.4.1

**Comment by PPPxiaomoer** [comment]
How to draw loss and prec@1?
